computation-local-test/src/test/resources/com/powsybl/config/test/config.yml

```diff
@@ -0,0 +1,2 @@
+computationDocker:
```

Comment:
maybe snake-case instead, for more consistency?

Reply:
Done